hbase-client/src/test/java/org/apache/hadoop/hbase/client/TestOperation.java

```diff
@@ -60,6 +59,9 @@
 import org.apache.hadoop.hbase.testclassification.SmallTests;
 import org.apache.hadoop.hbase.util.BuilderStyleTest;
 import org.apache.hadoop.hbase.util.Bytes;
+import org.apache.hadoop.hbase.util.GsonUtil;
+import org.apache.hbase.thirdparty.com.google.common.reflect.TypeToken;
```

Comment:
while working on HBASE-23052 I realized this is the wrong `TypeToken`. should be `com.google.gson.reflect.TypeToken`

Reply:
It has something to do with jdk7 compatibility or some issue in general?
Sure, Let me take care of this.

Reply:
just in general. the common.reflect package is from Guava. gson.reflect is from GSON. both just happen to be available in the hbase-thirdparty-miscellaneous artifact.

it only came up in the jdk7 stuff because a couple of the options made it so relocated guava wasn't around anymore.